transpire/internal/config.py

```diff
@@ -195,3 +195,22 @@ def from_cwd(cls, cwd: Path = Path.cwd()) -> "ClusterConfig":
                 "cluster.toml not found up to current git or fs boundary"
             )
         return cls.from_cwd(cwd.parent)
+
+def get_config(name: str, module_name: str | None = None, cwd: Path = Path.cwd()) -> ModuleType:
```

Comment:
`module_name` is always equal to `name`, so there's no need to have two arguments -- just have `name: str | None = None`

Reply:
Got it!